packages/flutter/lib/src/foundation/serialization.dart

```diff
@@ -30,33 +30,33 @@ class WriteBuffer {
   }
 
   /// Write a Uint16 into the buffer.
-  void putUint16(int value) {
-    _eightBytes.setUint16(0, value, Endian.host);
+  void putUint16(int value, {Endian endian}) {
```

Comment:
The default value should be specified here instead of using "?? Endian.host" below. And there should be an assert that endian is not null.

Reply:
I've tried with `Endian.host` but ended with error "The default value of an optional parameter must be constant.dart(non_constant_default_value)"

![carbon (2)](https://user-images.githubusercontent.com/7928961/71854207-6e6f8580-3118-11ea-97b8-82f390d4d27c.png)